### PR TITLE
support customize user & group key in jwt

### PR DESCRIPTION
**Pull request by ponytailer**
Token is fixed,  maybe need direct the user key when use the `create_jwt_token`.

thanks.
#40

### CODE REVIEW

File: ratelimit/auths/jwt.py [hunk 1]
```diff
@@ -18,6 +18,9 @@
 def create_jwt_auth(
     key: Union[bytes, str, RSAPublicKey, RSAPrivateKey],
     algorithms: Union[List[str], str],
+    user_key: str = "user",
+    group_key: str = "group",
+    default_group: str = "default"
```
Comment (abersheeran) [inline ratelimit/auths/jwt.py:23]: Group default value always must be `default`. Please remove the parameter.

**Review by abersheeran — CHANGES_REQUESTED** [review]

File: ratelimit/auths/jwt.py (hunk 1 above)
Comment (ponytailer) [reply ratelimit/auths/jwt.py:23]: OK

**Comment by ponytailer** [comment]
```
Run black --check --diff .
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.9/x64/bin/black", line 5, in <module>
    from black import patched_main
  File "/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/black/__init__.py", line 52, in <module>
    from typed_ast import ast3, ast27
  File "/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/typed_ast/ast3.py", line 40, in <module>
    from typed_ast import _ast3
ImportError: /opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/site-packages/typed_ast/_ast3.cpython-39-x86_64-linux-gnu.so: undefined symbol: _PyUnicode_DecodeUnicodeEscape
Error: Process completed with exit code 1.
```

can retry ?

**Comment by abersheeran** [comment]
Very strange actions problem, I will try to fix it later.

**Comment by abersheeran** [comment]
Found it. https://github.com/python/typeshed/pull/6278 

Can you execute `poetry update` and submit the new poetry lock to this PR?

**Comment by ponytailer** [comment]
> 

OK, I push the new `poetry.lock`

**Comment by abersheeran** [comment]
Thank you for your efforts!

**Pull request merged**